invenio_app_ils/schemas/items/item-v1.0.0.json

```diff
@@ -102,6 +106,21 @@
       "type": "string",
       "title": "The persistent identifier of this Item"
     },
+    "price": {
+      "properties": {
+        "currency": {
+          "title": "Currency of the price (Vocabulary)",
+          "type": "string"
+        },
+        "value": {
+          "minimum": 0,
+          "title": "Price value",
+          "type": "number"
+        }
+      },
+
```

Comment:
I think price should require both value and currency

```
      "required": [
        "currency",
        "value"
      ],
```

Reply:
If it is required and the user doesn't change the default currency in the form there is an error `bad request` because of currency field missing. This happens also in order's form.